solara/server/starlette.py

```diff
@@ -230,9 +230,9 @@ async def receive(self):
                 fut = self.portal.spawn_task(self.ws.receive)
 
             message = await asyncio.wrap_future(fut)
-        if "text" in message:
+        if "text" in message and message["text"] is not None:
```

Comment:
Fwiw
`if message.get("text") is not None`

Reply:
yeah, I agree